src/app/modules/features/components/profile/financial-information/financial-information.component.ts

```diff
@@ -42,7 +43,9 @@ export class FinancialInformationComponent implements OnInit, OnDestroy {
   }
 
   getCardInfo() {
+    // console.log(this.profileService.userProfile$.subscribe())
```

Comment:
Remove the comment if they are no longer used

Reply:
thanks... onit